src/server/command/worldedit/set.zig

```diff
@@ -0,0 +1,53 @@
+const std = @import("std");
+
+const main = @import("main");
+const User = main.server.User;
+const Pattern = @import("Pattern.zig");
+
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+
+pub const description = "Set all blocks within selection to a block.";
+pub const usage = "/set <pattern>";
+
+pub fn execute(args: []const u8, source: *User) void {
+	if(args.len == 0) {
+		source.sendMessage("#ff0000Missing required <pattern> argument.", .{});
+		return;
+	}
+	const pos1 = source.worldEditData.selectionPosition1 orelse {
+		return source.sendMessage("#ff0000Position 1 isn't set", .{});
+	};
+	const pos2 = source.worldEditData.selectionPosition2 orelse {
+		return source.sendMessage("#ff0000Position 2 isn't set", .{});
+	};
+
+	var seed: u64 = @as(u64, @intCast(pos1[0])) ^ (@as(u64, @intCast(pos1[1])) << 15) ^ (@as(u64, @intCast(pos1[2])) << 31);
```

Comment:
This code breaks for negative coordinates.
Also you could just work with `main.seed`, to get a different starting seed every time.
```
const seed = &main.seed;
```
(also there is main.random.initSeed3D, in case you actually want to have the same seed for each position)

Reply:
I think it makes more sense to have it always random, despite my initial intentions. Then you can re-do the thing to get different pattern if you don't like it first time.

Reply:
I think it will also be nice to add optional --seed/-s flag to specify seed manually and add print for current seed, but that should be separate PR.

Reply:
I don't think that makes sense. If you want the exact same structure, then you can just copy paste it.
And for other purposes the seed is kind of useless anyways.

Reply:
You may want to change size of the selection but keep the pattern. Although, since this is just noise there shouldn't be anything that special about that pattern. I guess we scrap that idea.